### PR TITLE
Compound borrows

### PR DESCRIPTION
**Pull request by yabirgb**
This PR introduces decoders for:

- Repay events in compound
- Borrow events in compound
- Collect comp token when repaying
- Changes default log level for VCR

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/compound/decoder.py [hunk 1]
```diff
@@ -122,6 +124,49 @@ def _decode_redeem(
...
+    def _decode_borrow_and_repay(
+            self,
+            tx_log: EvmTxReceiptLog,
+            decoded_events: list[HistoryBaseEntry],
+            compound_token: EvmToken,
+    ) -> tuple[Optional[HistoryBaseEntry], list[ActionItem]]:
+        """
+        Decode borrow and repayments for compound tokens
+        """
+        underlying_token = get_crypto_asset_by_symbol(
```
Comment (nebolax) [inline rotkehlchen/chain/ethereum/modules/compound/decoder.py:136]: Won't it be better to check `underlying_tokens` of the compound token?

**Review by nebolax — COMMENTED** [review]
Also "Changes default **debug level** for VCR" confused me :stuck_out_tongue: 

It looked like the log level should have been changed to debug, but in fact it is changed to critical :)

File: rotkehlchen/chain/ethereum/modules/compound/decoder.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/chain/ethereum/modules/compound/decoder.py:136]: The reason is that for example the CUSDC token (0x39AA39c021dfbaE8faC545936693aC917d5E7563) doesn't have USDC as underlying token in our database and we use the symbol in the token to know what token is the underlying one. We can change it but I would do in another PR. Also not 100% sure if they have a registry or something to keep it up to date

Comment (nebolax) [reply rotkehlchen/chain/ethereum/modules/compound/decoder.py:136]: Yeah, okay, checking underlying tokens is just cleaner, but the current approach should be okay too I guess.

The only question is whether cETH can appear here? If yes, then the cast to evm token that is happening below will be wrong.

File: rotkehlchen/tests/external_apis/test_coingecko.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def test_asset_data(session_coingecko):
         identifier='yearn-finance',
         symbol='yfi',
         name='yearn.finance',
-        image_url='https://assets.coingecko.com/coins/images/11849/small/yfi-192x192.png?1598325330',  # noqa: E501
+        image_url='https://assets.coingecko.com/coins/images/11849/small/yearn-finance-yfi.png?1676174746',  # noqa: E501
```
Comment (nebolax) [inline rotkehlchen/tests/external_apis/test_coingecko.py:47]: This is already fixed

Comment (LefterisJP) [reply rotkehlchen/tests/external_apis/test_coingecko.py:47]: yes but it's in bugfixes and this is a develop PR. It's either this or merge bugfixes to develop, which I want to do but well ... takes time

**Review by LefterisJP — APPROVED** [review]
LGTM. I assume the events are decoded as they should be, since you tested it and added tests

**Pull request merged**